WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -817,4 +817,21 @@ def create
     end
     params.require(:registration).permit(*permitted_params)
   end
+
+  private def registration_from_params
+    Registration.find(params[:id])
```

Comment:
Is this an insecure implementation? Because I'm not "whitelisting" certain parameters using `allow`

Reply:
That notion of `allow`ing certain parameters is merely a Rails ideology and not directly related to security.
You could do `params.require(:id)` which makes it so that an error is thrown when no id is passed (which makes sense because without an ID the entire route doesn't even resolve)

Reply:
Since you're not accessing any other fields, you're not getting into hot waters.
The `allow` mantra is only relevant when you pass the params hash as a whole into a SQL search function, for example.